### DISCUSSION TITLE
Synology nas Audiobookshelf "refused to connect". Looking for help.

### DISCUSSION THREAD

**Discussion by Baldbloke** [advplyr/audiobookshelf#1406] [Q&A]
Hi there clever peeps,

I'm trying to install Audiobookshelf on my 220+ Synology nas.

I've followed the steps here: https://neellik.com/how-to-install-audiobookshelf-on-your-synology-nas-with-docker/

Tried running the task as root and also as my user name on the nas. Same results when I try to access the audiobookshelf page.

When I try to access Audiobookshelf from my pc or my tablet at my Synology local IP which is 192.168.1.107:13378 it comes up with error "refused to connect".

My UID and GID appear to be correct. I've checked them by SSH'ing in to my nas and typed "id".

I enabled email notifications on the task created and received the email which states:
 "Standard output/error:
docker: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create?name=audiobookshelf": dial unix /var/run/docker.sock: connect: permission denied.
See 'docker run --help"

I've had a shuftie at the Docker help pages but due to my lack of knowledge, experience and plain old lack of intellegence, I'm stumped.

Any ideas please?

**Comment by Weldawadyathink** [comment]
That is a really really weird way to run docker on a Synology. I would recommend not using that website. Just use a generic guide to use docker on Synology, along with the guide on Audiobookshelf.com. 

My guess is docker is not installed, or the task runner is running as a user that does not have permission to use docker. Install docker from the Synology App Store and use the docker gui to install Audiobookshelf.

**Comment by Baldbloke** [comment]
Thank you Weldawadyathink

I persevered. All my fault (no surprise there). I was incorrectly entering the path for audiobooks. Once I correctly enabled email reporting I could then see the error notifications. It is up and running now,
Many thanks
